spec/features/manage_cmr/subscriptions/edit_subscriptions_spec.rb

```diff
@@ -99,13 +95,15 @@
       end
 
       it 'takes the user to the show page and has the correct data' do
-        expect(page).to have_content(@new_name)
-        expect(page).to have_content(@subscription['Query'])
-        expect(page).to have_content(@c_ingest_response['concept-id'])
+        expect(page).to have_content('Test_edit_subscriptions_122')
```

Comment:
You can use the variable to compare the page content instead of the hard coded values.